addons/spreadsheet/static/src/pivot/pivot_functions.js

```diff
@@ -72,6 +72,11 @@ const ODOO_PIVOT = {
         assertPivotsExists(pivotId, this.getters);
         assertMeasureExist(pivotId, measure, this.getters);
         assertDomainLength(domainArgs);
+        const dataSource = this.getters.getPivotDataSource(pivotId);
+        const loadingError = dataSource.getLoadingError();
```

Comment:
@pro-odoo what do you think about this?
I'm adding this to the generic data source but it won't necessarily makes sense for spreadsheet pivot

Reply:
indeed it does not make sense, and generic data source will not be available in spreadsheet anyway :/
For this version of course it's ok.
For later, I have an idea but not sure it's a good one :D : override the compute function of PIVOT in odoo to add this check